config/crd/transformer_config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: TransformerConfig
+nameReference:
+- version: v1
+  kind: Service
+  fieldSpecs:
+  - group: apiextensions.k8s.io
+    version: v1
+    kind: CustomResourceDefinition
+    path: spec/conversion/webhook/clientConfig/service/name
+namespace:
+- group: apiextensions.k8s.io
+  version: v1
+  kind: CustomResourceDefinition
+  path: spec/conversion/webhook/clientConfig/service/namespace
+  create: false
```

Comment:
I kept these files because I think it's a good idea to have a validating webhook for our CRDs in the near future. I can remove them for now and readd when necessary.

Reply:
I wouldn't add it until we need them

Reply:
I've removed them. We can apply this patch when we need them again: https://github.com/kedacore/http-add-on/compare/4aae719471b064f1c9d23a36f3d5b3621bdf68aa..7293f8cef92f284c79b28484a1d58c865417e967.